ProgramFiles/Utilities/curvature_mode_toolbox/illustrate_backbone_shapespace.m

```diff
@@ -14,18 +28,29 @@ function illustrate_backbone_shapespace(curvdef,paramvalues,orientation)
     error('Unsupported curvature definition')
 end
 
+% Specify plot number as 171 unless specified otherwise
+if ~exist('plotnum','var')
+    plotnum = 171; % 171 was the hardcoded default before I modified this
+end
+
 % Get the gradient of parameter values in the first two dimensions
 [~,grid_spacing_x] = gradient(paramvalues{1});
 [grid_spacing_y,~] = gradient(paramvalues{2});
 
-fh = figure(171);
+% Create the figure
+fh = figure(plotnum);
 close(fh);
-fh = figure(171);
-
+fh = figure(plotnum);
+set(gcf,'name',orientation);
 
 axh = axes('Parent',fh);
 axis equal
-hold on
+hold on 
```

Comment:
Would probably be good to have these explicitly target the axis:

axis(axh,'equal')
hold(axh,'on')

etc. for xticks, yticks, box

Reply:
Done! I used the newer ``axh.YTick = ...`` approach for xticks, yticks, and box. I think it's more readable, but I can change it to a ``set(axh, 'YTick', ...)`` if you want.